### DISCUSSION TITLE
Add failures

### DISCUSSION THREAD

**Discussion by epiclashy** [magknight/787aviatorsTracker#576] [Ideas]
Add some failures, even just basic ones like engine fires, cargo fires, and electrical system failures would be good. Also simulate things like the engine fire extinguisher system and emergency oxygen like the ff 767 does.

**Comment by Simleybob** [comment]
Add pressurization to that list, like in the zibo you could cause a pressure failure and so you press the oxygen mask it will 'give' you oxygen, simulated. Then you would need to do any emergency decent etc.

**Comment by aeroniemi** [comment]
We don't support, and have no intention to support, failure/non-standard scenarios, it is outside of the scope of the project.
